frontend/src/locales/fr.json

```
@@ -300,6 +307,14 @@
       "uppercase": "Une lettre majuscule requise."
     }
   },
+  "statistic": {
+    "activeUsers": "Membres actifs",
+    "deletedUsers": "Membres supprimés",
+    "totalGradidoAvailable": "GDD total en circulation",
+    "totalGradidoCreated": "GDD total épuisé",
```

Épuisé bedeutet erschöpft! Soll aber wohl geschöpft heißen oder? Wenn ja dann:
```suggestion
    "totalGradidoCreated": "GDD total puisé",
```